xy/point_centroid.go

```diff
@@ -16,6 +16,20 @@ func PointsCentroid(point *geom.Point, extra ...*geom.Point) geom.Coord {
 	return calc.GetCentroid()
 }
 
+// MultiPointCentroid computes the centroid of the multi point argument
+//
+// Algorithm: average of all points
+func MultiPointCentroid(point *geom.MultiPoint) geom.Coord {
```

Comment:
It is the average of all points (in multi point)  I will add the clarification to the comment